osu.Game/OsuGame.cs

```diff
@@ -886,7 +886,9 @@ public bool OnPressed(GlobalAction action)
                     return true;
 
                 case GlobalAction.ToggleGameplayMouseButtons:
-                    LocalConfig.Set(OsuSetting.MouseDisableButtons, !LocalConfig.Get<bool>(OsuSetting.MouseDisableButtons));
+                    var mouseDisableButtons = LocalConfig.GetBindable<bool>(OsuSetting.MouseDisableButtons);
+
+                    mouseDisableButtons.Value = !mouseDisableButtons.Value;
```

Comment:
can be shortened with the toggle method in `BindableBool`:
```suggestion
                    ((BindableBool)LocalConfig.GetBindable<bool>(OsuSetting.MouseDisableButtons)).Toggle();
```

Reply:
I prefer the original logic because it doesn't depend on an internal type that `ConfigManager` returns. If there's ever a time that `ConfigManager` doesn't return a `BindableBool` as a `Bindable<bool>`, then this will silently fail.

Reply:
Good catch, honestly feels like there should be certain extensions class providing that method, instead of inside a `BindableBool` class, but that's for another day.

Reply:
For reference, how does the silent fail occur? On a quick glance it looks like it'd throw on cast failure?

Reply:
Probably meant if returned instance type changed framework-side, then this code will be forgotten about and someone will raise an issue at some point saying their game crashes when toggling this action. That's what I understood.